### PR TITLE
Functional tests - Search hook in Positions page

### PR DESCRIPTION
**Pull request by nesrineabdmouleh**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | 1.7.7.x
| Description?  | Search hook in Positions page
| Type?         | refacto
| Category?     | TE
| BC breaks?    | no
| Deprecations? | no
| Fixed ticket? | no
| How to test?  | TEST_PATH="functional/BO/08_design/05_positions/02_searchHook.js" URL_FO=yourShopURL npm run specific-test

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/20468)

### CODE REVIEW

File: tests/UI/pages/BO/design/positions/index.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+class Positions extends BOBasePage {
+  constructor() {
+    super();
+
+    this.pageTitle = 'Positions';
+
+    // Selectors
+    this.searchInput = '#hook-search';
+    this.modulePositionForm = '#module-positions-form';
+    this.searchResultHookNameSpan = `${this.modulePositionForm} section[style] header span span`;
```
Comment (boubkerbribri) [inline tests/UI/pages/BO/design/positions/index.js:13]: Can't found any class for or something like that ?

Comment (SimonGrn) [inline tests/UI/pages/BO/design/positions/index.js:13]: That's such a beautiful selector... can we do something about it ? :laughing:

**Review by SimonGrn — APPROVED** [review]

File: tests/UI/pages/BO/design/positions/index.js (hunk 1 above)
Comment (SimonGrn) [reply tests/UI/pages/BO/design/positions/index.js:13]: Hey ! I'm the first !

Comment (nesrineabdmouleh) [reply tests/UI/pages/BO/design/positions/index.js:13]: the class is the same for all hooks, the only difference is in the style or the child ID :/

Comment (nesrineabdmouleh) [reply tests/UI/pages/BO/design/positions/index.js:13]: Yes, we can add a specific class for each hook, we will fix it in develop branch :)

Comment (boubkerbribri) [reply tests/UI/pages/BO/design/positions/index.js:13]: we will add a class but we will do it on develop since we do not touch twigs on 1.7.7.x

**Review by boubkerbribri — APPROVED** [review]

**Review by Progi1984 — APPROVED** [review]

**Pull request merged**